.gitignore

```diff
@@ -20,3 +20,66 @@ yarn-error.log*
 
 # fastlane
 fastlane/report.xml
+
+# custom
+ios/Pods
+android/maven
+
```

Comment:
The Pods and maven files was not in the .gitignore generated by react-native init, though I looked through some react native examples on github and from the projects I saw, they did not have the Pods / maven folders in git